packages/wallet/frontend/src/utils/crypto.ts

```diff
@@ -0,0 +1,22 @@
+import cryptolib from 'crypto'
+
+const secretKeyHex =
+  '00112233445566778899aabbccddeeff00112233445566778899aabbccddeeff'
+const iv = 'rafiki-testnet23'
+
+const encrypt = (text: string): string => {
```

Comment:
I'm not sure about this approach. Since the `encrypt` function is used on the client-side it will also have `secretKeyhex` and `iv` in the final client bundle.